packages/code-snippet/src/CodeSnippetWidget.tsx

```diff
@@ -367,6 +367,7 @@ export class CodeSnippetWidget extends ReactWidget {
           <button
             className={CODE_SNIPPETS_HEADER_BUTTON_CLASS}
             onClick={this.addCodeSnippet.bind(this)}
+            title="Add Code Snippet"
```

Comment:
I think "Add" might be a bit too vague, maybe "Add New" or "Create New" instead

Reply:
agree! updated title